README.md

```diff
@@ -1,3 +1,16 @@
+## It's so good to be here and see all your smiling faces! :blue_heart::yellow_heart: 
+
+```javascript
```

Comment:
Hmmm, are you sure we need JS in the .md file, and it'll work here?

Reply:
Markdown will output javascript as a block of code (but plain text) because this line - ```javascript - gives us only syntax highlighting. If there's no language indicated, there's no syntax highlighting :)

Reply:
Cool addition! Though, it should be 
```js
const user = {...
```
;)

Also, you can just use `js` instead of `javascript`.

Reply:
That's my bad... Yes, I agree with you entirely :) Thank you!